TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -0,0 +1,18 @@
+{
+ 
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc3-*"
+        },
+        "Newtonsoft.Json": "9.0.1-beta1"
```

Comment:
fixed.

---
In reply to: [66352854](https://github.com/dotnet/core-setup/pull/97#discussion_r66352854) [](ancestors = 66352854,66351645,66351095,66346538)